menuconfig.py

```diff
@@ -165,100 +165,276 @@
 view the help of the selected item without leaving the dialog.
 """[1:-1].split("\n")
 
-def _init_styles():
-    global _PATH_STYLE
-    global _SEPARATOR_STYLE
-    global _LIST_STYLE
-    global _LIST_SEL_STYLE
-    global _LIST_INVISIBLE_STYLE
-    global _LIST_INVISIBLE_SEL_STYLE
-    global _HELP_STYLE
-    global _DIALOG_FRAME_STYLE
-    global _DIALOG_BODY_STYLE
-    global _DIALOG_EDIT_STYLE
-    global _JUMP_TO_EDIT_STYLE
-    global _INFO_TEXT_STYLE
+# Default menuconfig theme
+DEFAULT_STYLE = \
```

Comment:
Should have a single initial `_`, to mark it as internal. That hides it in generated documentation, and when people do `from menuconfig import *` (though that's unlikely).

The only public interface in this module so far is the `menuconfig()` function, so all other global variables/functions get a leading `_`. Local variables never get a leading `_` though.